querybook/webapp/components/DataDocListsModal/DataDocListsModal.tsx

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+import { useSelector } from 'react-redux';
+
+import { EmptyText } from 'ui/StyledText/StyledText';
+import { Modal } from 'ui/Modal/Modal';
+import { BoardDetailedList } from 'components/BoardDetailedList/BoardDetailedList';
+import { currentEnvironmentSelector } from 'redux/environment/selector';
+import { Loading } from 'ui/Loading/Loading';
+import { BoardResource } from 'resource/board';
+import { IDataDoc } from 'const/datadoc';
+import { useResource } from 'hooks/useResource';
+
+interface IProps {
+    dataDoc: IDataDoc;
+    onHide: () => void;
+}
+
+export const DataDocListsModal: React.FunctionComponent<IProps> = ({
+    dataDoc,
+    onHide,
+}) => {
+    const environment = useSelector(currentEnvironmentSelector);
+
+    const { data: boards, isLoading } = useResource(
+        React.useCallback(
+            () =>
+                BoardResource.getItemBoards(
+                    environment.id,
+                    'data_doc',
+                    dataDoc.id
+                ),
+            [environment.id, dataDoc.id]
+        )
+    );
+
+    if (isLoading) {
+        return <Loading />;
```

Comment:
Modal?

Reply:
Changed to remove this & returned loading below